packages/browser/src/lib/is-thenable.ts

```diff
@@ -2,7 +2,7 @@
  *  Check if  thenable
  *  (instanceof Promise doesn't respect realms)
  */
-export const isThenable = (value: unknown): boolean =>
+export const isThenable = (value: unknown): value is Promise<unknown> =>
```

Comment:
Unfortunately, we don’t actually know if this is a Promise from this —- we know it’s a thenable. PromiseLike<unknown> might be closer, but I vaguely remember some issue  that stopped me from making this a type guard.

Reply:
What do you recommend? Revert this change altogether?

Reply:
I might just create another function called isPromise that does the same thing you want, but also checks for finally and catch

Reply:
I'll revert it.

Reply:
```suggestion
export const isThenable = (value: unknown): boolean =>
```

Reply:
Although can someone explain what does this mean "instance of Promise doesn't support realms"?

cc @chrisradek